### PR TITLE
fixed bugs and added filters

### PR DESCRIPTION
**Pull request by erdemkoral**
fixed bugs such as;
changed images source,
change html file name to index,
moved jquery library script tag,
fixed extarnal link url.

Added filters, so articles can be filtered by chicken name and temperament.
Added teaser function, so user can view more or less content.

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
-  $newChicken.find('li:nth-child(3)').text("Starts Laying Eggs By: " + this.startLaying);
-  $newChicken.find('li:last').text("Average Weight: " + this.weight);
+  $newChicken.find('#1').text("Egg Production: " + this.eggProduction);
+  $newChicken.find('#2').text("Temperament: " + this.temperament);
+  $newChicken.find('#3').text("Starts Laying Eggs By: " + this.startLaying);
+  $newChicken.find('#4').text("Average Weight: " + this.weight);
   $newChicken.find('.publishedOn').text("Published on " + this.publishedOn +" (" + daysAgo +" days ago)");
   $newChicken.find('.description').html(this.description);
-  $newChicken.find('.headShot').html(this.image);
+  $newChicken.find('.head-shot img').attr('src',  this.image);
```
Comment (JPGothro) [inline app/app.js:27]: Watch your eslint to check on correct spacing and indenting... on this line, only one space after "'src',".

**Comment by erdemkoral** [comment]
Got it. Thanks..

File: app/app.js [hunk 2]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+}
+
+function populateTemperFilter (chickenTemper){
+  var optionTag = '<option value="' + chickenTemper+ '">' + chickenTemper + '</option>';
+  if($('.temper-filter option[value="' + chickenTemper + '"]').length === 0) {
+    $('.temper-filter').append(optionTag);
+  }
+}
+
 allChickens.forEach(function(a){
```
Comment (JPGothro) [inline app/app.js:51]: It's better practice to move this function near the bottom of this code file.

File: app/app.js [hunk 3]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+    var selectedValue = $(this).val();
+    console.log(selectedValue);
+    if(selectedValue){
+      $('article[data-name="' + selectedValue + '"]').fadeIn(800);
+    }else{
+      $('article').not('.template').show();
+    }
+  });
+}
+chicFilter();
```
Comment (JPGothro) [inline app/app.js:70]: Better practice to move these function invocations to the bottom of the file...

File: app/app.js [hunk 4]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+// filter by title
+function chicFilter(){
+  $('.chicken-filter').change(function(){
+    $('article').hide();
+    var selectedValue = $(this).val();
+    console.log(selectedValue);
+    if(selectedValue){
+      $('article[data-name="' + selectedValue + '"]').fadeIn(800);
+    }else{
+      $('article').not('.template').show();
```
Comment (JPGothro) [inline app/app.js:66]: You aren't resetting the other filter box when this one is being used...

File: app/app.js [hunk 5]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+  $('.temper-filter').change(function(){
+    $('article').hide();
+    var selectedValue = $(this).val();
+    console.log(selectedValue);
+    if(selectedValue){
+      $('article[data-temper="' + selectedValue + '"]').fadeIn(800);
+    }else{
+      $('article').not('.template').show();
+    }
+  });
```
Comment (JPGothro) [inline app/app.js:83]: You aren't resetting the other filter when this is being used.

File: app/app.js [hunk 6]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+}
+tempFilter();
```
Comment (JPGothro) [inline app/app.js:85]: Better to move this function invocation to the end of the file...

**Comment by erdemkoral** [comment]
I am working on it now.

File: app/app.js [hunk 7]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
-});
-$('nav').find('li:first').on('click', function(){
-  $('#chickens').hide();
-  $('#about').fadeIn().css({"height": "600px"})
-});
-$('nav').find('li:nth-child(2)').on('click', function(){
-  $('#about').hide();
-  $('#chickens').fadeIn()
+function teaser(){
+  console.log('in the fuction');
```
Comment (JPGothro) [inline app/app.js:88]: Don't forget to remove console logs when they are no longer of use.

File: app/app.js [hunk 8]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+   if($(this).text() === 'Read more') {
+    $('.description').find('p').fadeIn();
+     $(this).text('Show Less');
+   } else if ($(this).text() === 'Show Less'){
+     hiddenContent.hide();
+     $(this).text('Read more');
+   }
+ });
+}
+teaser();
```
Comment (JPGothro) [inline app/app.js:103]: see note about moving this to the bottom of the file.

File: app/app.js [hunk 9]
```diff
@@ -16,38 +16,103 @@ Chicken.prototype.toHtml = function() {
...
+
+$(document).ready(function(){
+  //show only About section
+  $('nav').find('li:first').on('click', function(){
+    $('#chickens').hide();
+    $('#about').fadeIn().css({"height": "600px"});
+  });
```
Comment (JPGothro) [inline app/app.js:110]: No prevent default in this click.
Also, can you see a way to make these two click handlers actually work for both?

I noticed that when I clicked on these, the filters for the articles were still showing. Do you know why?

**Pull request merged**